sunspec/core/modbus/mbmap.py

```diff
@@ -22,12 +22,17 @@
 """
 
 import struct
+import sys
 
 try:
     import xml.etree.ElementTree as ET
 except:
     import elementtree.ElementTree as ET
 
+# Python 3 compatibility for long()
+if sys.version_info > (3,):
+    long = int
```

Comment:
I'm confused.  I am not aware of there needing to be any agreement between the object and the format string when it comes to int vs long other than simply having enough bytes specified in the format string to hold the actual value being packed into it.  Maybe you could write a test that fails when using `int` instead of `long` to help me understand?

Here's an example running in py2 where for the format string 'L' you get the same sequence of bytes regardless of specifying `int` or `long`.  https://repl.it/JWvD/1

Just to be direct, I'm not suggested varying the format strings, I'm suggesting exclusively using `int` and not using `long` at all since `int` objects will elevate to `long` if needed (see example in above link).